### DISCUSSION TITLE
Bounty Master marker in Velen not showing.

### DISCUSSION THREAD

**Discussion by Sovietude** [Aelto/W3_RandomEncounters_Tweaks#197] [Q&A] [answered]
Love the mod, thank you for your work!

The issue I have is Bounty Master does not have a marker, not on the map, not on the mini map in Velen. I have found him at one the locations as shown at RER bible and was able to fully interact with him, but he didn't have a marker on him. (forgot to screenshot it)

The weird thing is marker works fine in White orchard when I started my playthrough and when I fast travel there from Velen ( I do that to sell trophies).
I haven't been to Novigrad yet. Is it possible that I have to naturally visit all spawn locations for a marker to appear? but then why marker wasn't there when Bounty Hunter was right in front of me.
I use W3EE Redux with all latest versions.

**Comment by Aelto** [comment, marked as answer]
Hello! Happy to hear you're enjoying the mod, thank you!

The fix for the marker issue missing from velen is already implemented and will come in the next release that i hope to release this weekend if i've got the time!
